packages/experiments/src/components/PersonaCoin/PersonaCoin.test.tsx

```diff
@@ -8,6 +8,9 @@ import { PersonaTestImages } from '@uifabric/experiments/lib/common/TestImages';
 import { Icon, Image, Text } from 'office-ui-fabric-react';
 import { setRTL } from '../../Utilities';
 
+// Disable "ban" rule which gives spurious errors about .find()
```

Comment:
There are **A LOT** more test files which need this comment (waiting to add them to the PR to avoid making things hard to read and triggering tons of codeowners).